api/machine/machine.proto

```diff
@@ -28,6 +28,8 @@ service MachineService {
       returns (EtcdLeaveClusterResponse);
   rpc EtcdForfeitLeadership(EtcdForfeitLeadershipRequest)
       returns (EtcdForfeitLeadershipResponse);
+  rpc GenerateConfiguration(GenerateConfigurationRequest)
```

Comment:
should it be `ConfigGenerate` like `ServiceList` ? 

or we want it align with `ApplyConfiugration`?

Reply:
yeah, I wanted to align with `ApplyConfiguration`, but I'm fine with any name for it.

Reply:
`GenerateConfiguration` reads better IMHO.